docs/guide/service/annotations.md

```diff
@@ -13,42 +13,43 @@
     These annotations are specific to the kubernetes [service resources reconciled](#lb-type) by the AWS Load Balancer Controller. Although the list was initially derived from the k8s in-tree `kube-controller-manager`, this
     documentation is not an accurate reference for the services reconciled by the in-tree controller. 
 
-| Name                                                                                             | Type                    | Default                   | Notes                                                  |
```

Comment:
Nit: can we fix the formatting issue by only add a new line instead of change existing lines.